### ISSUE TITLE
Very slow training on CPU

### ISSUE THREAD

**Issue by chrisspen** [HarryVolek/PyTorch_Speaker_Verification#61]
I'm training with the CPU backend for Torch with Python3.7 on anaconda with a quad-core CPU, and it seems very slow. After running for 12 hours, it had only completely 30 epoches out of the max of 950, and my loss had only gotten down to about 12. I estimate a full training would likely take an entire week. Is this normal without cuda? Is there anything I can do to speed up performance without getting a GPU?

**Comment by MahmoudAliEng** [comment]
Hi @chrisspen ,
I had the same issue here, did you found a solution ? Could you please tell me how did you solve it?

**Comment by fmaker** [comment]
Yes, that is normal with big models. You need a GPU.
